src/ramprakash/narcissisticNumbers.ts

```diff
@@ -0,0 +1,38 @@
+export const isNarcissistic = (num: number): boolean => {
+  if (num <= 0) {
+    return false;
+  }
+
+  if(num > 0 && num < 10) {
+    return true;
+  }
+
+  
+  //Find the length of the given number
+  let divisor: number = 1, numLength: number = 0;
+  while(num / divisor > 1){
```

Comment:
`>`? .what if it is equal to `1`. This logic fails if the input number is multiple of 10. For example with input `100` it will respond with length as `2`. Better if we find any other way than home cooking length implementation of a number.